src/test/java/com/puppycrawl/tools/checkstyle/api/AutomaticBeanTest.java

```diff
@@ -97,6 +97,14 @@ public void testContextualize_ConversionException() {
         }
     }
 
+    @Test(expected=IllegalStateException.class)
+    public void testTestBean() {
+        final TestBean testBean = new TestBean();
+        testBean.setVal(0);
+        testBean.setWrong("wrongVal");
+        testBean.setExceptionalMethod("someValue");
+    }
+
     private static class TestBean extends AutomaticBean {
```

Comment:
Cannot be suppressed. Details https://github.com/checkstyle/checkstyle/pull/1823